
            <!DOCTYPE html>
            <html lang="en">
                <head>
                    <meta charset="UTF-8" />
                    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
                    <title>Schema Evaluator - Individual Analysis</title>
                </head>
                <body style="background-color: #f6f6f6">
                    <noscript> You need to enable JavaScript to run this app. </noscript>
                    <div id="root"></div>
                    <script> const _preComputedDataInd = {"data": {"allele_sizes": [318, 318, 318, 318, 318], "alleles_median": 318, "alleles_mode": 318, "locus_ids": ["GCF-000011665-protein1101_1", "GCF-000011665-protein1101_S_SAMN08690957.contigs.fa_24/03/2021T12:15:59_2", "GCF-000011665-protein1101_S_SAMEA4550588.contigs.fa_25/03/2021T02:09:15_3", "GCF-000011665-protein1101_S_SAMN07154407.contigs.fa_26/03/2021T10:46:11_4", "GCF-000011665-protein1101_S_SH0264A.contigs.length-GCcontent-kmerCov.mappingCov.polished.fasta_03/04/2021T17:14:28_5"], "nr_alleles": 5, "size_range": "318-318"}, "locus_name": "GCF-000011665-protein1101"} </script>
                    <script> const _exceptions = [] </script>
                    <script> const _cdsDf = {"Alleles not multiple of 3": 0, "Alleles shorter than 201 nucleotides": 0, "Alleles w/ >1 stop codons": 0, "Alleles wo/ Start/Stop Codon": 0, "CDS": 5, "Gene": "GCF-000011665-protein1101.fasta", "Missing Allele IDs": ["None"], "Number of alleles": 5, "Total Invalid Alleles": 0, "coding_strand": "Not provided", "contig": "Not provided", "genome": "Not provided", "name": "Not provided", "start": "Not provided", "stop": "Not provided", "url": "Not provided"} </script>
                    <script> const _msaData = {"sequences": [{"name": "1", "sequence": "MSEKEFFKQLKALHLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILELYTRKKDLDVEGKIETFLTEQHLFFDKVETYIDTEQLYQVAFHFAF"}, {"name": "2", "sequence": "MSEKEFFKQLKALHLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILELYTRKKDLDVEGKIETFLTEQHLFFDKVETYIDTEQLYQVAFHFAF"}, {"name": "3", "sequence": "MNEKEFFKQLKALNLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILELYTRKKDLYVEEKIEAFLMEQHLFFDKVETYIDTEQLYQVAFHFAF"}, {"name": "4", "sequence": "MNEKEFFKQLKALNLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILELYTRKKDLYVEEKIEAFLMEQHLFFDKVETYIDTEQLYQVAFHFAF"}, {"name": "5", "sequence": "MSEKEFFKQLKALHLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILELYTRKKDLDVEGKIETFLTEQHLFFDKVETYIDTEQLYQVAFHFAF"}]} </script>
                    <script> const _phyloData = {"phylo_data": "(((\n1_1\n:0.00000,\n2_2\n:0.00000):0.00000,\n5_5\n:0.00000):0.20917,(\n3_3\n:0.00000,\n4_4\n:0.00000):0.20917);\n"} </script>
                    <script> const _minLen = 201 </script>
                    <script> const _fasta = `>1
MSEKEFFKQLKALHLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILEL
YTRKKDLDVEGKIETFLTEQHLFFDKVETYIDTEQLYQVAFHFAF
>2
MSEKEFFKQLKALHLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILEL
YTRKKDLDVEGKIETFLTEQHLFFDKVETYIDTEQLYQVAFHFAF
>3
MNEKEFFKQLKALNLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILEL
YTRKKDLYVEEKIEAFLMEQHLFFDKVETYIDTEQLYQVAFHFAF
>4
MNEKEFFKQLKALNLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILEL
YTRKKDLYVEEKIEAFLMEQHLFFDKVETYIDTEQLYQVAFHFAF
>5
MSEKEFFKQLKALHLPIAYHHFEEGHSPSPPFMVYYSPQSENFGADNQVYHKGAQFILEL
YTRKKDLDVEGKIETFLTEQHLFFDKVETYIDTEQLYQVAFHFAF
` </script>
                    <script src="./main_ind.js"></script>
                </body>
            </html>
            