
            <!DOCTYPE html>
            <html lang="en">
                <head>
                    <meta charset="UTF-8" />
                    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
                    <title>Schema Evaluator - Individual Analysis</title>
                </head>
                <body style="background-color: #f6f6f6">
                    <noscript> You need to enable JavaScript to run this app. </noscript>
                    <div id="root"></div>
                    <script> const _preComputedDataInd = {"data": {"allele_sizes": [366, 366, 366, 366, 366], "alleles_median": 366, "alleles_mode": 366, "locus_ids": ["GCF-005163865-protein920_1", "GCF-005163865-protein920_S_GCF-000630025.1-Stre-pyog-ABC020015285-V1.fasta_16/11/2020T14:21:59_2", "GCF-005163865-protein920_S_GCF-009887425.1-ASM988742v1.fasta_16/11/2020T14:22:44_3", "GCF-005163865-protein920_S_GCF-009887505.1-ASM988750v1.fasta_16/11/2020T14:22:44_4", "GCF-005163865-protein920_S_GCF-900982695.1-11791-7-73.fasta_16/11/2020T14:23:02_5"], "nr_alleles": 5, "size_range": "366-366"}, "locus_name": "GCF-005163865-protein920"} </script>
                    <script> const _exceptions = [] </script>
                    <script> const _cdsDf = {"Alleles not multiple of 3": 0, "Alleles shorter than 201 nucleotides": 0, "Alleles w/ >1 stop codons": 0, "Alleles wo/ Start/Stop Codon": 0, "CDS": 5, "Gene": "GCF-005163865-protein920.fasta", "Missing Allele IDs": ["None"], "Number of alleles": 5, "Total Invalid Alleles": 0, "coding_strand": "Not provided", "contig": "Not provided", "genome": "Not provided", "name": "Not provided", "start": "Not provided", "stop": "Not provided", "url": "Not provided"} </script>
                    <script> const _msaData = {"sequences": [{"name": "1", "sequence": "MENRYRTNLKKVFLTDQELATLNDCIGQSRCKNFSQYARRVLLNPNMAFLTIDTSNYQELVFELRRIGNNINQIARTVNQTHILTPTDRDNLCQGISQLIKEVEKDFDIKCQQLRAFYGSD"}, {"name": "2", "sequence": "MVYRYRTNLKKVFLTDSELHQLNERIDKSHCQNFSVYARKVLLNPNMSFVTINTDTYDQLVFELRRIGNNINQIARAINQSHLISQEQLQELSKGVSELITGVEKEFQVEVKRLKELYGSH"}, {"name": "3", "sequence": "MENRYRTNLKKVFLTDQELANLNNHISQSSCQNFSAYARKVLLNPNMTFLTIDTSNYQDLVFELRRIGNNINQIAKTVNQTKLLTVEQRDNLKTGIGQLIAEVDKDFYVKCQKLREFYGSH"}, {"name": "4", "sequence": "MVYRYRTNLKKVFLTDSELYQLNERIAKSHCQNFSVYARKVLLNPNMSFVTINTDTYDQLVFELRRIGNNINQIARAINQSHLISQKQLQELSKGVGELIKEVDKEFQVEVKRLKEFHGSH"}, {"name": "5", "sequence": "MVYRYRTNLKKVFLTDSELHQLNERVAKSHCQNFSVYARRVLLNPNMSFVSLNTDAYDQLVFELRRIGNNINQIARAINQSHLISQEQLQELSKGVSELITEVEKEFQVEVKKLKEFHGSH"}]} </script>
                    <script> const _phyloData = {"phylo_data": "((\n1_1\n:0.43180,\n3_3\n:0.43180):0.04484,((\n2_2\n:0.13347,\n5_5\n:0.13347):0.01256,\n4_4\n:0.14603):0.33061);\n"} </script>
                    <script> const _minLen = 201 </script>
                    <script> const _fasta = `>1
MENRYRTNLKKVFLTDQELATLNDCIGQSRCKNFSQYARRVLLNPNMAFLTIDTSNYQEL
VFELRRIGNNINQIARTVNQTHILTPTDRDNLCQGISQLIKEVEKDFDIKCQQLRAFYGS
D
>2
MVYRYRTNLKKVFLTDSELHQLNERIDKSHCQNFSVYARKVLLNPNMSFVTINTDTYDQL
VFELRRIGNNINQIARAINQSHLISQEQLQELSKGVSELITGVEKEFQVEVKRLKELYGS
H
>3
MENRYRTNLKKVFLTDQELANLNNHISQSSCQNFSAYARKVLLNPNMTFLTIDTSNYQDL
VFELRRIGNNINQIAKTVNQTKLLTVEQRDNLKTGIGQLIAEVDKDFYVKCQKLREFYGS
H
>4
MVYRYRTNLKKVFLTDSELYQLNERIAKSHCQNFSVYARKVLLNPNMSFVTINTDTYDQL
VFELRRIGNNINQIARAINQSHLISQKQLQELSKGVGELIKEVDKEFQVEVKRLKEFHGS
H
>5
MVYRYRTNLKKVFLTDSELHQLNERVAKSHCQNFSVYARRVLLNPNMSFVSLNTDAYDQL
VFELRRIGNNINQIARAINQSHLISQEQLQELSKGVSELITEVEKEFQVEVKKLKEFHGS
H
` </script>
                    <script src="./main_ind.js"></script>
                </body>
            </html>
            