
            <!DOCTYPE html>
            <html lang="en">
                <head>
                    <meta charset="UTF-8" />
                    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
                    <title>Schema Evaluator - Individual Analysis</title>
                </head>
                <body style="background-color: #f6f6f6">
                    <noscript> You need to enable JavaScript to run this app. </noscript>
                    <div id="root"></div>
                    <script> const _preComputedDataInd = {"data": {"allele_sizes": [213, 213, 213, 213, 213, 213], "alleles_median": 213, "alleles_mode": 213, "locus_ids": ["GCF-002844355-protein966_1", "GCF-002844355-protein966_S_GCF-002250265.1-ASM225026v1-genomic.fna_23/03/2021T15:05:12_2", "GCF-002844355-protein966_S_GCF-900983815.1-19944-6-173-genomic.fna_23/03/2021T15:06:00_3", "GCF-002844355-protein966_S_GCF-900985705.1-8898-3-37-genomic.fna_23/03/2021T15:06:17_4", "GCF-002844355-protein966_S_GCF-900988945.1-8849-2-20-genomic.fna_23/03/2021T15:06:44_5", "GCF-002844355-protein966_S_GCF-900993675.1-19944-8-106-genomic.fna_23/03/2021T15:07:25_6"], "nr_alleles": 6, "size_range": "213-213"}, "locus_name": "GCF-002844355-protein966"} </script>
                    <script> const _exceptions = [] </script>
                    <script> const _cdsDf = {"Alleles not multiple of 3": 0, "Alleles shorter than 201 nucleotides": 0, "Alleles w/ >1 stop codons": 0, "Alleles wo/ Start/Stop Codon": 0, "CDS": 6, "Gene": "GCF-002844355-protein966.fasta", "Missing Allele IDs": ["None"], "Number of alleles": 6, "Total Invalid Alleles": 0, "coding_strand": "Not provided", "contig": "Not provided", "genome": "Not provided", "name": "Not provided", "start": "Not provided", "stop": "Not provided", "url": "Not provided"} </script>
                    <script> const _msaData = {"sequences": [{"name": "1", "sequence": "MVKEVKKLADDMACLVIWAIILLLLHLLTSGWIWLAVKFVMTVMLILGVRDWFVDWTLFSLKQKYYKTGE"}, {"name": "2", "sequence": "MVKEVKKLADDMACLVIWAIILLLLYLLTSGWIWLAVKFVMTVMLIFGVRDWFVDWTLFSLKQKYYKTDE"}, {"name": "3", "sequence": "MVKEVKKLADDMACLVIWAIILLLLHLLTSSWIWLAVKFVMTVMLILGVRDWFVDWTLFSLKQKYYKTGE"}, {"name": "4", "sequence": "MVKEVKKLADDMAFLVIWAIIFLFLHLFTSGWIWLAVKFVMTVMLILGVRDWFVDWTLFSLKQKYYKTGE"}, {"name": "5", "sequence": "MVKEVKKLADDMACLVFLGFILLLLHLLTSGWIWLAVKFVMTVMLILGVRDWFVDWTLFSLKQKYYKTGE"}, {"name": "6", "sequence": "MVKEVKKLADDMACLVIWAIILLLLHLLISGWIWLAVKFVMTVMLILGVRDWFVDWTLFSLKQKYYKTGE"}]} </script>
                    <script> const _phyloData = {"phylo_data": "(((((\n1_1\n:0.00000,\n3_3\n:0.00000):0.08368,\n6_6\n:0.08368):0.04603,\n5_5\n:0.12971):0.07312,\n2_2\n:0.20283):0.05440,\n4_4\n:0.25723);\n"} </script>
                    <script> const _minLen = 201 </script>
                    <script> const _fasta = `>1
MVKEVKKLADDMACLVIWAIILLLLHLLTSGWIWLAVKFVMTVMLILGVRDWFVDWTLFS
LKQKYYKTGE
>2
MVKEVKKLADDMACLVIWAIILLLLYLLTSGWIWLAVKFVMTVMLIFGVRDWFVDWTLFS
LKQKYYKTDE
>3
MVKEVKKLADDMACLVIWAIILLLLHLLTSSWIWLAVKFVMTVMLILGVRDWFVDWTLFS
LKQKYYKTGE
>4
MVKEVKKLADDMAFLVIWAIIFLFLHLFTSGWIWLAVKFVMTVMLILGVRDWFVDWTLFS
LKQKYYKTGE
>5
MVKEVKKLADDMACLVFLGFILLLLHLLTSGWIWLAVKFVMTVMLILGVRDWFVDWTLFS
LKQKYYKTGE
>6
MVKEVKKLADDMACLVIWAIILLLLHLLISGWIWLAVKFVMTVMLILGVRDWFVDWTLFS
LKQKYYKTGE
` </script>
                    <script src="./main_ind.js"></script>
                </body>
            </html>
            