
            <!DOCTYPE html>
            <html lang="en">
                <head>
                    <meta charset="UTF-8" />
                    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
                    <title>Schema Evaluator - Individual Analysis</title>
                </head>
                <body style="background-color: #f6f6f6">
                    <noscript> You need to enable JavaScript to run this app. </noscript>
                    <div id="root"></div>
                    <script> const _preComputedDataInd = {"data": {"allele_sizes": [492, 492, 492, 492, 492], "alleles_median": 492, "alleles_mode": 492, "locus_ids": ["GCF-005161605-protein228_1", "GCF-005161605-protein228_S_GCF-002268555.1-ASM226855v1-genomic.fna_23/03/2021T15:40:56_2", "GCF-005161605-protein228_S_GCF-009901545.1-ASM990154v1-genomic.fna_23/03/2021T15:41:17_3", "GCF-005161605-protein228_S_GCF-009905915.1-ASM990591v1-genomic.fna_23/03/2021T15:41:17_4", "GCF-005161605-protein228_S_SAMN02570759.contigs.fa_24/03/2021T13:40:13_5"], "nr_alleles": 5, "size_range": "492-492"}, "locus_name": "GCF-005161605-protein228"} </script>
                    <script> const _exceptions = [] </script>
                    <script> const _cdsDf = {"Alleles not multiple of 3": 0, "Alleles shorter than 201 nucleotides": 0, "Alleles w/ >1 stop codons": 0, "Alleles wo/ Start/Stop Codon": 0, "CDS": 5, "Gene": "GCF-005161605-protein228.fasta", "Missing Allele IDs": ["None"], "Number of alleles": 5, "Total Invalid Alleles": 0, "coding_strand": "Not provided", "contig": "Not provided", "genome": "Not provided", "name": "Not provided", "start": "Not provided", "stop": "Not provided", "url": "Not provided"} </script>
                    <script> const _msaData = {"sequences": [{"name": "1", "sequence": "MDQEQIFDRGARVTLETGKQLFQFLAYAAQQLYKTYDEHKQIGEQSFKNFNQSSLTKDHIDFLEADVNLSKFTDELKKSGVRFAFKDLGDGTKQVWFEAPNREVIADALRHTLNEIVNDPKKAKEKYMKSEQELTPKEQINKIKKATKEKLDTVKTKKKGKSI"}, {"name": "2", "sequence": "MDQEQIFDRGARVTLETGKQLFQFLAYAAQQLYKTYDEHKQIGEQSFKNFNQSSLTKDHIDFLEADVNLSKFTDELKKSGVRFAFKDLGDSTKQVWFEAPNREVIADALRHTLNEIVNDPKKAKEKYMKSEQELTPKEQINKIKKATKEKLDTVKTKKKGKSI"}, {"name": "3", "sequence": "MDQEQIFDRGARVTLETGKQLFQFLAYATQQLYKAYDEQKQMGQQSWKNFNQLSLTKDHIDFLEADVNLKKFTDELKKSGVRFAFKDLGDGTKQVWFEAPNKEVIADALRHTLNEIVNDPKKAKEKYMKSEKKLTPKEQINKIKKATKEKLDTVKTKKKGKSI"}, {"name": "4", "sequence": "MDQEQIFDRGARVTLETGKQLFQFLAYAAQQLYKTYDEHQQIGEQSFKNFNQSSLTKDHIDFLEADVNLRKFTDELKKSGVRFAFKDLGDGTKQVWFEAPNREVIADALRHTLNEIVNDPKKAKEKYMKSEKELTPKEQINKIKKATKEKLDTVKTKKKGKSI"}, {"name": "5", "sequence": "MDQEQIFDRGARVTLETGKQLFQFLAYAAQQLYKAYDEQKQIGQQSFKKLNQSSLTKDHIDFLEADVNLRKFTDELKKSGVRFAFKDLGDGTKQVWFEAPNRGVIADALSHTLNEIINDPKKAKEKYMKSEKELTPKEQINKIKKATKEKLDTVKTKKKGKSI"}]} </script>
                    <script> const _phyloData = {"phylo_data": "((((\n1_1\n:0.00000,\n2_2\n:0.00000):0.10414,\n4_4\n:0.10414):0.00752,\n3_3\n:0.11166):0.04236,\n5_5\n:0.15402);\n"} </script>
                    <script> const _minLen = 201 </script>
                    <script> const _fasta = `>1
MDQEQIFDRGARVTLETGKQLFQFLAYAAQQLYKTYDEHKQIGEQSFKNFNQSSLTKDHI
DFLEADVNLSKFTDELKKSGVRFAFKDLGDGTKQVWFEAPNREVIADALRHTLNEIVNDP
KKAKEKYMKSEQELTPKEQINKIKKATKEKLDTVKTKKKGKSI
>2
MDQEQIFDRGARVTLETGKQLFQFLAYAAQQLYKTYDEHKQIGEQSFKNFNQSSLTKDHI
DFLEADVNLSKFTDELKKSGVRFAFKDLGDSTKQVWFEAPNREVIADALRHTLNEIVNDP
KKAKEKYMKSEQELTPKEQINKIKKATKEKLDTVKTKKKGKSI
>3
MDQEQIFDRGARVTLETGKQLFQFLAYATQQLYKAYDEQKQMGQQSWKNFNQLSLTKDHI
DFLEADVNLKKFTDELKKSGVRFAFKDLGDGTKQVWFEAPNKEVIADALRHTLNEIVNDP
KKAKEKYMKSEKKLTPKEQINKIKKATKEKLDTVKTKKKGKSI
>4
MDQEQIFDRGARVTLETGKQLFQFLAYAAQQLYKTYDEHQQIGEQSFKNFNQSSLTKDHI
DFLEADVNLRKFTDELKKSGVRFAFKDLGDGTKQVWFEAPNREVIADALRHTLNEIVNDP
KKAKEKYMKSEKELTPKEQINKIKKATKEKLDTVKTKKKGKSI
>5
MDQEQIFDRGARVTLETGKQLFQFLAYAAQQLYKAYDEQKQIGQQSFKKLNQSSLTKDHI
DFLEADVNLRKFTDELKKSGVRFAFKDLGDGTKQVWFEAPNRGVIADALSHTLNEIINDP
KKAKEKYMKSEKELTPKEQINKIKKATKEKLDTVKTKKKGKSI
` </script>
                    <script src="./main_ind.js"></script>
                </body>
            </html>
            